CMakeLists.txt

```diff
@@ -527,6 +527,22 @@ endif(BUILD_SPARSE)
 
 if(BUILD_GRAPHBOLT)
   message(STATUS "Configuring graphbolt library")
+  if(CMAKE_SYSTEM_NAME MATCHES "Linux")
```

Comment:
is it better to move it from `BUILD_GRAPHBOLT` so that other DGL module could also leverage this lib one day?

Reply:
Sure, I have moved it outside.